server/app/views/applicant/EditTiClientView.java

```diff
@@ -81,24 +82,69 @@ public Content render(
       optionalSuccessMessage = Optional.empty();
     }
     boolean isSuccessfulSave = tiClientInfoForm.isPresent() && !tiClientInfoForm.get().hasErrors();
-    HtmlBundle bundle =
-        layout
-            .getBundle(request)
-            .setTitle(title)
-            .addMainContent(
-                renderHeader(tiGroup.getName(), "py-12", "mb-0", "bg-gray-50"),
-                hr(),
-                renderSubHeader(pageHeader).withId(pageId).withClass("my-4"),
-                renderBackLink(),
-                renderSuccessAlert(isSuccessfulSave, successToast, optionalSuccessMessage),
-                requiredFieldsExplanationContent(),
-                renderAddOrEditClientForm(
-                    tiGroup, optionalAccountModel, request, tiClientInfoForm, messages))
-            .addMainStyles("px-20", "max-w-screen-xl");
+    HtmlBundle bundle;
+    if (isSuccessfulSave && applicantIdOfNewlyAddedClient.isPresent()) {
```

Comment:
The main content for the two is different. One with form the other without form. Can you share an example of how to split the .addMainContent() method?

Reply:
It would look something like this (example is without parameters):

```
private DivTag renderContent(some parameters) {
    boolean isSaved = isSuccessfulSave && applicantIdOfNewlyAddedClient.isPresent();
    return div()
      .condWith(isSaved,
        renderSuccessAlert(),
        renderApplicationsStartButton(),
        renderBackToClientListButton()
      )
      .condWith(!isSaved, 
        requiredFieldsExplanationContent(),
        renderAddOrEditClientForm()
      );
 }
 ```